examples/gallery/ui/pages/controls_page.slint

```diff
@@ -30,6 +30,14 @@ export component ControlsPage inherits Page {
                 primary: true;
             }
 
+            Button {
+                text: @tr("colorize-icon");
```

Comment:
As text on the icon that's odd (and let's be honest, this can't be translated). I understand that if the text of the button is too long, it will overflow.

I suggest to replace the single row of buttons (the horizontal layout) with a grid layout that has two rows, and in the second row you could place this button but with the text "Button With Colorized Icon". The text could also be something else, but it should be consistent with the text of the other buttons.

Reply:
Thank you for the feedback. I have changed that. I used Vertical and Horizontal Layout instead because I want to left align the buttons.